specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -1694,6 +1694,190 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/restorePoints": {
+      "get": {
+        "tags": [
+          "ObjectReplicationRestorePoints"
+        ],
+        "operationId": "ObjectReplicationRestorePoints_List",
+        "description": "List the object replication restore points associated with the storage account.",
+        "x-ms-examples": {
+          "StorageAccountListObjectReplicationRestorePoints": {
+            "$ref": "./examples/StorageAccountListObjectReplicationRestorePoints.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- List object replication restore points successfully.",
+            "schema": {
+              "$ref": "#/definitions/ObjectReplicationRestorePoints"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/objectReplicationPolicies/{objectReplicationPolicyId}/restorePointMarkers/default": {
+      "put": {
+        "tags": [
+          "ObjectReplicationRestorePointMarkers"
+        ],
+        "operationId": "ObjectReplicationRestorePointMarkers_Create",
+        "description": "Create the object replication restore point marker of the storage account.",
```

Comment:
The rest API doc is autogenerated from swagger, so it's better to add more information about the API in the description, like: 
1. In which scenario user need use "object replication restore point" feature
2. What the effect after run this API.
3. Any other facts can help user choose to use this API or not.

Reply:
reverted this change from PR it is not in scope for this release.